### PR TITLE
[DNM] CEM-15500 WebUI modify and extend Status Field in CR

### PR DESCRIPTION
**Pull request by katrybacka**

### CODE REVIEW

**Comment by katrybacka** [comment]
recheck ha

File: test/e2e/ha/core_contrail_services_test.go [hunk 1]
```diff
@@ -424,30 +425,32 @@ func getHACluster(namespace string) *contrail.Manager {
...
-					{Name: "init", Image: "registry:5000/common-docker-third-party/contrail/python:" + versionMap["python"]},
-					{Name: "nodeinit", Image: "registry:5000/contrail-nightly/contrail-node-init:" + versionMap["cemVersion"]},
-					{Name: "redis", Image: "registry:5000/common-docker-third-party/contrail/redis:" + versionMap["redis"]},
-					{Name: "webuijob", Image: "registry:5000/contrail-nightly/contrail-controller-webui-job:" + versionMap["cemVersion"]},
-					{Name: "webuiweb", Image: "registry:5000/contrail-nightly/contrail-controller-webui-web:" + versionMap["cemVersion"]},
-				},
-			},
-		},
-	}
+	//TODO uncomment when WebUI will support environment without Keystone
```
Comment (vijayrap) [inline test/e2e/ha/core_contrail_services_test.go:428]: Please remove the commented code

**Comment by katrybacka** [comment]
I'm going to close this PR and merge R2008 branch to master. The requested change is already in R2008

**Comment by katrybacka** [comment]
It will be merged with a merge of R2008 to master

**Pull request closed without merging**